tests/python/contrib/test_hexagon/test_autotvm.py

```diff
@@ -143,12 +143,13 @@ def test_autotvm(hexagon_session):
             ),
         ),
     }
-    target_hexagon = tvm.target.hexagon("v68")
     task = autotvm.task.create(
-        "demo_template", args=[], target=target_hexagon, target_host=target_hexagon
+        "demo_template",
+        args=[],
+        target=get_hexagon_target("v68"),
     )
     tune_tasks([task], **options)
 
 
 if __name__ == "__main__":
-    sys.exit(pytest.main(sys.argv))
+    tvm.testing.main()
```

Comment:
Side comment, thank you for updating these calls to use `tvm.testing.main()`.  I'd like to make a custom lint rule for it at some point, but there isn't one at the moment.

Reply:
yeah that would be great! I mentioned it to OSS team before.